mimic/text_generator.py

```diff
@@ -16,6 +16,11 @@ def __init__(self, model):
         """Initialize a TextGenerator with the given model type."""
         self.model = model
 
+    def train_from_zip(self, zip_file_path):
+        """Load the zip and initiate training by the model."""
+        text = load_text_zip(zip_file_path)
+        model.learn(text)
```

Comment:
Think that should be `text = self.load_text_zip(zip_file_path)` and `self.model.learn(text)`?